vrrpd/vrrp_northbound.c

```diff
@@ -40,12 +40,25 @@ static int lib_interface_vrrp_vrrp_group_create(struct nb_cb_create_args *args)
 	uint8_t version = 3;
 	struct vrrp_vrouter *vr;
 
-	if (args->event != NB_EV_APPLY)
-		return NB_OK;
-
-	ifp = nb_running_get_entry(args->dnode, NULL, true);
+	ifp = nb_running_get_entry(args->dnode, NULL, false);
```

Comment:
nit: ideally you should call this function with the last parameter set to `true` during the NB_EV_APPLY phase (not that it matters much...).

Reply:
got it